kolibri/plugins/learn/assets/src/views/PageHeader.vue

```diff
@@ -58,9 +58,9 @@
   }
 
   .progress-icon {
-    position: relative;
-    top: -2px;
     display: inline-block;
+    float: right;
+    margin-top: -2px;
```

Comment:
similar here - maybe we should extend `KLabeledIcon` to support leading and trailing icons? (We've had a need for that in other places too)

Reply:
Also note that from #5662

> the title should be right-aligned with the exercise icon on the right and the progress icon on the left. (the English title should be a locally LTR inline block within an outer RTL wrapper)

It's a bit subtle but I'm fairly sure that's the expected behavior here

Reply:
So add a 'text-align: left' rule to the outer container? I feel like there's probably a lot of other headers in the app that don't have this rule and are always left-aligned in RTL mode.

Reply:
Actually most headers do act properly. I wonder where PageHeader is being aligned left in spite of RTL

Reply:
I removed the wrapping div from PageHeader, so the top-level tag is h1 and is properly transformed when in RTL mode.

Reply:
a couple references on mixed RTL/LTR behavior:

* https://www.w3.org/International/articles/inline-bidi-markup/
* https://www.w3.org/International/articles/inline-bidi-markup/uba-basics